### ISSUE TITLE
Search textfield is not working

### ISSUE THREAD

**Issue by anubhavpulkit** [amahi/ios#332]
## Description:
The search text field of the navbar is not taking input. Inside shares when we try to tap on the search text field then Sort options pop-up.

## Screen Recording:
![video-to-gif-converter](https://user-images.githubusercontent.com/47811606/88705130-bc87bd00-d12c-11ea-942c-9aca14708e58.gif)

**Comment by anubhavpulkit** [comment]
This bug is Only present in SE 2nd generation device.

**Comment by cpg** [comment]
It is available in newer devices.

It is in fact a horrible implementation at this point. There are at least five or more bugs related to this:

- unresponsive search box -- tapping on it yields no keyboard at all (nothing) or the sort selector
- responsive box, but only in landscape mode
- duplication or triplication of the search icon at the top right, each time a rotation happens, one new icon appears, it seems
- appearance of a cancel button, or not. no idea how/when
- when able to type, the search is erratic, sometimes some titles are found, sometimes not

... and quite more ..

**Comment by anubhavpulkit** [comment]
Yes I found these `unresponsive search box and duplication or triplication of the search icon at the top right` when I'm testing in SE 2nd generation device after 2 or 3 times. But in other devices like XR, I can't find these issues.

**Comment by cpg** [comment]
Please list all issues here below, one by one.

Focus on fixing the first one.

Later, focus on fixing the second one.

**Comment by cpg** [comment]
Search works in landscape mode. That should be a very straighforward hint.

Working on colors, etc. means you are already not focused.

From my last message

- [ ] list all the issues one by one

**Comment by anubhavpulkit** [comment]
- [x] Unresponsive search box when tap first time(if we move to another UI controller and back again then it started working). Not responsive in any orientation.
- [x] Sometimes search bar worked as a sort button.
- [x] Duplication or triplication of the search icon at the top right in `Server file UI` only. 
- [x] The issue is present only in new devices(SE 2, iPhone 11 models)
- [ ] Appearance of a cancel button is quite random.
- [x] Foreground color of the text field is not changing according to different modes(always white color).
